introspection.go

```diff
@@ -348,6 +349,7 @@ func (c *Connection) IntrospectState(opts *IntrospectionOptions) ConnectionRunti
 		InboundExchange:  c.inbound.IntrospectState(opts),
 		OutboundExchange: c.outbound.IntrospectState(opts),
 		HealthChecks:     c.healthCheckHistory.asBools(),
+		LastActivity:     c.lastActivity.Load(),
```

Comment:
in the introspection, can we make this a `time.Time`, which will probably be easier for humans to read? Or could have 2 separate fields (one as nanos, one as human readable).

That will also allow the tests to compare the time directly.

Reply:
That was my first intuition as well, but it makes things weird because the reconstructed time.Time() doesn't have a timezone (Location), so direct comparison with a time.Date created in the test failed, even though they are identical.